reader/feed.py

```diff
@@ -9,32 +9,31 @@
 # Reader imports
 from reader import URL
 
-_CACHED_FEEDS = dict()  # type: Dict[str, feedparser.FeedParserDict]
+_CACHED_FEEDS: Dict[str, feedparser.FeedParserDict] = dict()
```

Comment:
We already have info about the data type, so probably using a `dict` literal is more straightforward and faster.

```suggestion
_CACHED_FEEDS: Dict[str, feedparser.FeedParserDict] = {}
```